dot/rpc/modules/system.go

```diff
@@ -75,6 +75,12 @@ func (sm *SystemModule) Name(r *http.Request, req *EmptyRequest, res *string) er
 	return nil
 }
 
+// ChainType returns the chain type
+func (sm *SystemModule) ChainType(r *http.Request, req *EmptyRequest, res *string) error {
```

Comment:
Can you add a struct for the response? I was thinking we should have a struct for all request and reponse.

Reply:
sure, will update!

Reply:
@arijitAD Why? I'm not disagreeing that all request and response should be a struct, but I want to better understand why.

Reply:
it seems like changing the types to a strict is what caused polkadot.js to break, so I think I will leave it as is for now

Reply:
I was thinking that it might make the code cleaner and in the future, we can just modify the struct if we want to add fields. But we can leave it for responses that have a single field.